### PR TITLE
Do not use gif

### PR DESCRIPTION
**Pull request by couet**
- Do not use gif output in tutorials. On machine where gif is not available is generated error when build the doxygen manual .
- use `auto`

### CODE REVIEW

File: tutorials/image/hsumanim.C [hunk 1]
```diff
@@ -69,10 +72,6 @@ void hsumanim() {
    slider->SetRange(0,1);
    total->Draw("sameaxis"); // to redraw axis hidden by the fill area
    c1->Modified();
-   // make infinite animation by adding "++" to the file name
-   if (gROOT->IsBatch()) c1->Print("hsumanim.gif++");
```
Comment (bellenot) [inline tutorials/image/hsumanim.C:73]: So no animated GIF anymore?

**Review by bellenot — APPROVED** [review]
I trust you, if all tests pass...

File: tutorials/image/hsumanim.C (hunk 1 above)
Comment (couet) [reply tutorials/image/hsumanim.C:73]: No, just the recipe on how to do it.

**Comment by hahnjo** [comment]
FYI: In basically all nightlies, `projectroot.runtutorials.tutorial_graphics_tmathtext` now fails, and this appears to be the first candidate...

**Comment by couet** [comment]
This PR https://github.com/root-project/root/pull/9604 should fix the problem

**Pull request merged**